charts/kong/crds/custom-resource-definitions.yaml

```diff
@@ -1,15 +1,75 @@
-# generated using: kubectl kustomize github.com/kong/kubernetes-ingress-controller/config/crd?ref=main
```

Comment:
We still use this to generate right? I think this was our only breadcrumb leading a reader back to how we actually include these CRDs in the chart, should we leave it?

Reply:
I forgot how file overwrites work. Whoops.